### PR TITLE
Clean-up setup scripts and data

### PR DESCRIPTION
**Pull request by BetsyMcPhail**
* Moved setup scripts and data files used by scripts to **scripts** subdirectory
* Removed generated data from repository
* Data is now generated in **files** subdirectory
* Removed unused files (scripts and data)
* Updated Generate Data documentation

### CODE REVIEW

**Comment by BetsyMcPhail** [comment]
PTAL @josephsnyder

File: Visual/scripts/pkgcsv_to_json.py [hunk 1]
```diff
@@ -11,20 +11,20 @@
 pkgAutocomple = set();
 
 def generate_packages_json():
-  pkgCatJson = json.load(open("PackageCategories.json", 'r'))
+  pkgCatJson = json.load(open("../PackageCategories.json", 'r'))
   pkgDesJson = json.load(open("PackageDes.json", 'r'))
   generate_output_json_dict(pkgCatJson, pkgDesJson)
-  with open("packages.json", 'w') as outputFile:
+  with open("../files/packages.json", 'w') as outputFile:
```
Comment (josephsnyder) [inline Visual/scripts/pkgcsv_to_json.py]: The index.php file still expects this to be in the same directory as it, not down in "files".

`d3.json("packages.json", function(json) {`

This applies to most of the json files that are generated.  We should either generate them at the above level or switch out where the d3 attempts to read them from.

Comment (BetsyMcPhail) [inline Visual/scripts/pkgcsv_to_json.py]: Thanks for the review. I can reproduce this on my system.

**Comment by BetsyMcPhail** [comment]
Can you please give this another try @josephsnyder ?

**Comment by BetsyMcPhail** [comment]
Updated to include the new BFF data generation process.

@josephsnyder can you PTAL?

**Comment by josephsnyder** [comment]
Still looks good to me.  No issues with the setup scripts or with the webpages on my local system

**Pull request merged**